Makefile

```
@@ -230,8 +230,22 @@ test: test-shell ## Runs unit tests with race detector and coverage (use -short
 	echo "Test coverage:"; \
 	go tool cover -func=coverage.out | tail -1
 
+# Listed before `test` so an unreadable threshold fails in seconds rather than
+# after the full race-enabled suite. Fails closed on both an empty value (yq
+# missing) and the literal "null" (yq prints that for a missing key, exit 0) --
+# bc would resolve either to 0 and silently pass the check below with no floor.
+# Keep in sync with the matching guard in .github/actions/go-coverage.
+.PHONY: check-coverage-threshold
+check-coverage-threshold:
```

checkmake (0.3.2)

[warning] Target body for "check-coverage-threshold" exceeds allowed length of 5 lines (6).

(maxbodylength)